js/bare_board.js

```diff
@@ -428,62 +298,11 @@ export class Board {
           rotation: d.rotation,
           frozen: d.frozen,
         })),
-      stock:                        this.stock.stock,
+      stock:                        this.stock ? this.stock.stock : {},  // hack for non-attached stock
       requiredDetectionProbability: this.level.requiredDetectionProbability,
```

Comment:
tak! 
chodzi o ty, by mieć bareBoard np do encyklopedii; bez level'u nie ma sensu (niekoniecznie to jest level do przejścia, moze tylko dane gdzie jaki tile jest); ale np. w owej encyklopedii nie ma stocka